test/webpack-build.test.js

```diff
@@ -237,4 +237,18 @@ for (const nodeVersion of SUPPORTED_NODE_VERSIONS) {
     });
     test.false(result.hasErrors());
   });
+
+  // Test that EJS modules can be packed because they are used by graphql
+  test.serial(`Can webpack modules that use .mjs modules when targetting ${nodeVersion}`, async (test) => {
```

Comment:
I would argue that this test case does not reflect our actual intent. In my opinion, our intent is to support `graphql` and EJS modules are an implementation detail. There are existing test cases that verify that GraphQL based Lambdas can be built and used.

Reply:
When I initially wrote this test nothing else in the project caused a failure. While I personally want graphql to work, from a general usability standpoint I think we should keep mjs files working even if grapqhl moves away from them.

Reply:
> from a general usability standpoint I think we should keep mjs files working even if grapqhl moves away from them.

I don't fully agree with this statement. However, this test case would ensure that if we decided to make a change to what we support that we would do so in a well managed way. So I think I agree with the conclusion that the test case should stay.